### ISSUE TITLE
Adding an appendix adds dots to chapter, section, figure, ... titles

### ISSUE THREAD

**Issue by rindPHI** [tudace/tuda_latex_templates#155]
I use the tudapub package to write my PhD thesis (great package by the way). Normally, chapters, sections etc. appear in the ToC as, e.g., "1 Introduction", figures have the caption "Figure 2.4.5 MyCaption" etc. But when I add an appendix, e.g. using the appendix package, like that:

\begin{appendices}
  \chapter{My First Appendix}
\end{appendices}

suddenly makes them appear with a trailing dot, e.g., "Figure 2.4.5. MyCaption". Appendices are listed as "A." etc. I load the appendix package with

\usepackage[title,toc]{appendix}

Adding the Koma script package option "noenddot" to the document class definition does not help. It is, by the way, not the fault of the appendix package, I also tried it without that package and the usual "\appendix" macro, which had the same effect.

Ah, and I'd rather not have these dots :) I'm grateful for help!

Thanks already & Best Regards,
Dominic

**Comment by TeXhackse** [comment]
Hey,
thanks for the kind words. 

Regarding your issue: Why you use the appendix package and not the KOMA-Script mechanism for that?

The appendix packages adds the enddots and kills the KOMA-Script mechanism for changing these. If there is no special reason for that package (and I currently can't imagine one) you should prefer using `\appendix`. If t's only about the extra toc entry, you can have a look at the tocbasic documentation for adding that as well. (though this one is usually not needed, because it's already visible due to the change in enumeration)

**Comment by rindPHI** [comment]
Well, I started using the appendix package because I though it'd fix the issue... Then I added the toc option because why not. It's actually not a super serious issue, just somebody proofreading commented on the dots and obviously found them distracting, so I tried to remove them, but since I managed to develop a 400p document, it takes up to 8 min with lualatex to compile it (other latex derivates, which used to be much faster but produced less nice results, completely refuse by now), I lost too much time and couldn't help but file this issue. I also tried to construct a MWE and nailed it down to the appendix, but I have to stop now unfortunately. So if nowbody has a quick-n-easy solution, never mind, and thanks very much!

**Comment by TeXhackse** [comment]
Okay, quick solution, just reset the option after loading appendix. The package might have a possibility to change that via option. If you want to know that, just check the documentation. But this works.
```
\usepackage[title,toc]{appendix}
\KOMAoptions{numbers=noendperiod}
```

**Comment by rindPHI** [comment]
Amazing, thanks, that works! I actually tried the option "noendperiod" (I did check the documentation before), but that did _not_ work. With your suggestion of setting the option via KOMAoptions after package inclusion, the dots magically disappear.

Thanks for the offer for help regarding compilation times. I'm using the subfiles package, so I do get quicker feedback for individual parts. I also invested a day in trying to get format files working, but that somehow was unsuccessful. What I'm doing additionally is using precompiled TikZ pictures, which helps a little. But it's true, for my use case, lualatex is really slow. Albeit produces better results. I'll submit end of the months, so no experiments now ;)
